app/models/PlayExampleProjects.scala

```diff
@@ -33,14 +32,14 @@ class ExamplesModule extends AbstractModule {
 @Singleton
 class PlayExampleProjectsService @Inject()(configuration: Configuration,
                                            ws: WSClient,
-                                           cache: CacheApi)(implicit ec: ExecutionContext) {
+                                           cache: SyncCacheApi)(implicit ec: ExecutionContext) {
   import scala.collection.JavaConverters._
 
   private val logger = org.slf4j.LoggerFactory.getLogger(this.getClass)
 
-  val templatesUrl = configuration.getString("examples.apiUrl").get
+  val templatesUrl: String = configuration.get[String]("examples.apiUrl")
 
-  val validPlayVersions: Set[String] = configuration.getStringList("examples.playVersions").get.asScala.toSet
+  val validPlayVersions: Set[String] = configuration.underlying.getStringList("examples.playVersions").asScala.toSet
```

Comment:
When I wrote the deprecation message I guess I was thinking about what the direct migration would be, if you still want to use `java.util.List` for some reason.